OpenRA.Game/Settings.cs

```diff
@@ -90,18 +90,36 @@ public class DebugSettings
 		public bool SanityCheckUnsyncedCode = false;
 		public int Samples = 25;
 		public bool IgnoreVersionMismatch = false;
+
+		[Desc("Dump performance data into cpu.csv and render.csv accessible via ~/.openra/Logs.")]
```

Comment:
Already did that as I promised https://github.com/OpenRA/OpenRA/pull/10133/files#diff-be79af2a24e3cc8e6022f6b41958d725R24